### DISCUSSION TITLE
Add support for long term statistics in Home Assistant

### DISCUSSION THREAD

**Discussion by Sjorsa** [teslamate-org/teslamate#1889] [Ideas]
Home Assistant just updated to 2021.8.0, which adds support for long terms statistics. 

It would be cool to be able to use the longer graphs in Home Assistant to quickly check long term trends.

More information can be found [here](https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics).

**Relevant entries from the logs**

```
paste logs here
```

**Screenshots**

**Data**

**Environment**

- TeslaMate version:
- Type of installation: Docker or Manual
- OS TeslaMate is installed on:
- User OS & Browser:
- Others:

**Comment by kimmilde** [comment]
Just add the sensors you want a Long term stats on in customize.yaml. Add state_class: measurement
